docs/how-to-guides/eagleye-integration-guide.md

```diff
@@ -0,0 +1,115 @@
+
+## What is Eagleye?
+
+Eagleye is an open-source GNSS/IMU-based localizer initially developed by [MAP IV. Inc](https://map4.jp/).
+Eagleye provides a cost-effective alternative to LiDAR and point cloud-based localization by using low-cost GNSS and IMU sensors to provide vehicle position, orientation, and altitude information.
+Using GNSS Doppler velocity, Eagleye can also compensate for wheel speed and IMU angular velocity bias to achieve more accurate speed and heading estimation.
+
+With the addition of Eagleye integration into Autoware, users will be able to choose between their existing LiDAR and point cloud-based localization stacks or GNSS/IMU-based Eagleye localizer, depending on their specific needs and operating environment.
+
+There are two ways to utilize Eagleye results in the Autoware localization stack:
+
+1. Feed only twist into the EKF localizer.
+
+   ![Eagleye twist integration](images/eagleye-integration-guide/eagleye_twist.drawio.svg)
+
+2. Feed both twist and pose from Eagleye into the EKF localizer (twist can also be used with regular `gyro_odometry`).
+
+   ![Eagleye pose twist integration](images/eagleye-integration-guide/eagleye_pose_twist.drawio.svg)
+
+Note that RTK positioning is only required for localization using the Eagleye pose. RTK positioning is not required for twist.
+
+## Behavior when Eagleye is executed by Autoware
+
+On startup, Eagleye will perform static and dynamic initialization.
+
+### Static initialization
+
+Eagleye needs to be stationary for about 3~5 seconds, according to the parameter `yaw_rate_offset_stop.estimated_interval` in `eagleye_config.yaml`, after startup for the yaw rate offset to be estimated.
+Static initialization is performed even in environments where GNSS is not received.
+
+### Dynamic initialization
+
+Next, it needs to travel in a straight line for about 20~30 seconds, as defined by the parameter `heading.estimated_minimum_interval` and `velocity_scale_factor.estimated_minimum_interval`, in order to estimate the wheel speed scale factor and azimuth angle. Once the dynamic initialization is completed, Eagleye will begin to output both twist and pose.
+
+## Eagleye setup
+
+### GNSS ROS driver setting
+
+In addition to latitude, longitude, and height information (`sensor_msgs/msg/NavSatFix`), Eagleye also requires velocity information from GNSS.
+The settings needed for each GNSS ROS driver are as follows:
+
+- [ublox_gps](https://github.com/KumarRobotics/ublox/tree/ros2/ublox_gps): This ROS driver publishes `sensor_msgs/msg/NavSatFix` and `geometry_msgs/msg/TwistWithCovarianceStamped` required by Eagleye with default settings. Therefore, no additional settings are required.
+- [septentrio_gnss_driver](https://github.com/septentrio-gnss/septentrio_gnss_driver/tree/ros2): Set `publish.navsatfix` and `publish.twist` in the config file [`gnss.yaml`](https://github.com/septentrio-gnss/septentrio_gnss_driver/blob/ros2/config/gnss.yaml#L90) to `true`
```

Comment:
This explanation gives reader the impression that Eagleye only works with these drivers. You might want to rewrite this so that readers can understand:
* requirements for the GNSS driver to use Eagleye is to publish `sensor_msgs/msg/NavSatFix` and `geometry_msgs/msg/TwistWithCovarianceStamped`.
* It is tested with the following example GNSS drivers: ublox_gps and septentrio_gnss_drivers

Reply:
Fixed